assets/src/edit-story/components/library/libraryProvider.js

```diff
@@ -35,13 +37,40 @@ import { ElementsPane, ElementsIcon } from './panes/elements';
 import { PageTemplatesPane, PageTemplatesIcon } from './panes/pageTemplates';
 import { getPaneId, Pane as SharedPane } from './panes/shared';
 
-const MEDIA = { icon: MediaIcon, Pane: MediaPane, id: 'media' };
-const MEDIA3P = { icon: Media3pIcon, Pane: Media3pPane, id: 'media3p' };
-const TEXT = { icon: TextIcon, Pane: TextPane, id: 'text' };
-const SHAPES = { icon: ShapesIcon, Pane: ShapesPane, id: 'shapes' };
-const ELEMS = { icon: ElementsIcon, Pane: ElementsPane, id: 'elements' };
+const MEDIA = {
+  icon: MediaIcon,
+  tooltip: __('WordPress media', 'web-stories'),
+  placement: TOOLTIP_PLACEMENT.BOTTOM_START,
+  Pane: MediaPane,
+  id: 'media',
+};
+const MEDIA3P = {
+  icon: Media3pIcon,
+  tooltip: __('Thirdparty media', 'web-stories'),
```

Comment:
"Third-party" perhaps?

Reply:
Forgot about this one - done in 5f2ba17.